### DISCUSSION TITLE
Slack integration for PostgreSQL connection

### DISCUSSION THREAD

**Discussion by riupie** [hoophq/hoop#552] [Q&A] [answered]
Based on the picture on this doc https://hoop.dev/docs/integrations/slack, we can review based on script that will be executed by user. I follow the guide on the doc but what happened is it only review and then accept the connection to postgres console not review what script that will be executed. How to limit user so it will review the script instead of the db connection?

**Comment by sandromello** [comment, marked as answer]
Hello,

This feature is only available in our enterprise plans, the open source version allows only time based review.
Contact us in the [support chat](https://help.hoop.dev/) if you want a free trial license to assess this feature.

**Reply by riupie** [reply]
I see, thanks!
